### ISSUE TITLE
Save graph data into Graph DB

### ISSUE THREAD

**Issue by RebeccaGreiner** [amosproj/amos2024ss05-knowledge-graph-extractor#25]
## User story
1. As a {}
2. I want / need {}
3. So that {}

## Acceptance criteria
* Take the valid graph format and save it into the Graph DB.

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
